src/database/migrations/20230418232949_AddAllowSharingAndLocationColumnsToParticipantsTable.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('participants', (table) => {
+    table.boolean('allowSharing').defaultTo(true);
```

Comment:
When there's a default column value, you usually also want to set that value for existing rows. I think that's probably a good idea here.